### ISSUE TITLE
Grammar for the question entries.

### ISSUE THREAD

**Issue by Kreijstal** [Kreijstal/diytest#3]
Well here we discuss how people would type/import questions into the website, this is extremely important to do right since it will be the core functionality of the website.

## How will we allow users to use LaTeX?
maybe we want to allow math questions and we also want to allow mathy answers, mathquill and all of that.
##How to implement multiple choice questions
what if the solutions is an image or an audio?

anyone has any questions?

**Comment by keerthivasan-r** [comment]
Yes, we can first do basic steps. Let us come up with a basic list of requirements for the prototype modal and start things off. we can incrementally improve the capabilities
